packages/account/src/Components/financial-details/financial-details-partials.tsx

```diff
@@ -166,10 +166,10 @@ const FinancialInformation = ({ employment_status }: TFinancialInformationProps)
             />
             {!shouldHideOccupationField(employment_status) && (
                 <FinancialDetailsOccupationDropdownField
-                    dropdown_list={getFormattedOccupationList(employment_status)}
+                    dropdown_list={getFormattedOccupationList(employment_status as TEmploymentStatus)}
```

Comment:
should we add some TODO in such places in our codebase, do refactor later types and avoid types assertions?
or it is fine to apply them like here?

Reply:
We could add TODOs. But for some values, we might have to typecast. For example string and enum.